test/backend_sum.in.cpp

```diff
@@ -535,4 +535,35 @@ NGRAPH_TEST(${BACKEND_NAME}, sum_stable_acc)
 
     EXPECT_TRUE(test::all_close_f(ref_results.at(0), bk_results.at(0), 24, 3));
 }
+
+NGRAPH_TEST(${BACKEND_NAME}, sum_stable_acc_double)
+{
+    std::string backend_name = "${BACKEND_NAME}";
+    if (backend_name == "INTERPRETER")
+    {
+        return;
+    }
+    Shape shape_a{10, 10, 20, 300};
+    auto A = make_shared<op::Parameter>(element::f64, shape_a);
+
+    Shape shape_rt{10};
+    auto f = make_shared<Function>(make_shared<op::Sum>(A, AxisSet{1, 2, 3}), ParameterVector{A});
+
```

Comment:
Just use a sequence known to fail if added sequentially and known to pass if added Kahnaentially.

Reply:
Add two simple test for float and double.